### ISSUE TITLE
Checkbox label text is not announced when hovering over label or input, with mouse tracking enabled

### ISSUE THREAD

**Issue by vanitabarrett** [nvaccess/nvda#13879]
### Steps to reproduce:

1. Open https://codepen.io/vanita-barrett/full/KKoVJRx in Chrome with NVDA version 2019.1 and above
2. Enable mouse tracking in NVDA (NVDA + m)
3. Hover over the checkbox label and the checkbox itself

Seems related to https://github.com/nvaccess/nvda/issues/11845

### Actual behavior:
NVDA does not announce the checkbox label when hovering over the label or the checkbox input, even with mouse tracking enabled

### Expected behavior:
NVDA should announce the checkbox label

Note: in version 2018.1 of NVDA, the checkbox label is announced when hovering over the checkbox input, but this doesn't happen in versions 2019.1 and above.

#### NVDA installed/portable/running from source:
Testing via https://assistivlabs.com/

#### NVDA version: 
Latest version 2022.1, but also present in all versions down to 2019.1 (and 2018.1, see comment in 'expected behaviour' section)

#### Windows version:
Windows 10 Pro

#### Name and version of other software in use when reproducing the issue: 
Chrome version 103

#### Other information about your system: 
N/A

#### Does the issue still occur after restarting your computer? 
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors. 
As noted above, the checkbox is not announced on hover of checkbox input and checkbox label from version 2019.1.
From 2018.1 - 2019.1, the checkbox is announced on hover of the checkbox input only (not the label).

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Unsure, been unable to try

**Comment by seanbudd** [comment]
Duplicate of #13843, #11845
See also #6722
